azure-pipelines/manifest.sh

```diff
@@ -31,29 +31,44 @@ DATE=$(cat .release_date | tr ' ' T | sed 's/TUTC/:00-00:00/')
 VERSION=$(cat .version)
 BASE="openttd-${VERSION}"
 
-echo "name: ${NAME}" >> manifest.yaml
+echo "name: ${NAME}" > manifest.yaml
 echo "date: ${DATE}" >> manifest.yaml
 echo "base: ${BASE}-" >> manifest.yaml
-echo "files:" >> manifest.yaml
 
 error=""
-for i in $(ls ${FOLDER} | grep -v ".txt$\|.md$\|sum$" | sort); do
-    if [ -n "$(echo $i | grep pdb.xz)" ]; then continue; fi
-    if [ -n "$(echo $i | grep dbg.deb)" ]; then continue; fi
 
-    if [ ! -e ${FOLDER}/$i.md5sum ] || [ ! -e ${FOLDER}/$i.sha1sum ] || [ ! -e ${FOLDER}/$i.sha256sum ]; then
-        echo "ERROR: missing checksum file for ${i}" 1>&2
-        error="y"
-        continue
-    fi
+FILES=files
+DEV_FILES=dev_files
```

Comment:
this makes the argument parsing a bit weird. Maybe just pass `files` & `dev_files` to `output_files` directly?

```bash
output_files dev_files $DEV_FILES
```